packaging/rpm-ostree.spec

```diff
@@ -224,7 +224,7 @@ sed -i -e '/https:\/\//d' cargo-vendor.txt
 %if %{without ostree_ext}
 rm -vrf $RPM_BUILD_ROOT/usr/libexec/libostree/ext
 %endif
-%if %{without kernel_install}
+%if %{with no_kernel_install}
 rm -vr $RPM_BUILD_ROOT/usr/lib/kernel/install.d
 %endif
```

Comment:
I think a clearer fix is to change this to be:

```
%if {with kernel_install}
%else
rm -vr $RPM_BUILD_ROOT/usr/lib/kernel/install.d
%endif
```

right? That's basically what I messed up.

Reply:
I am so confused already, I am doing a scratch build here to make sure it looks ok
: https://koji.fedoraproject.org/koji/taskinfo?taskID=128613318

Reply:
I think we might need:
```
%if %{without kernel_install}
%else
rm -vr $RPM_BUILD_ROOT/usr/lib/kernel/install.d
%endif
```

Reply:
https://koji.fedoraproject.org/koji/taskinfo?taskID=128614173 with the above

Reply:
updated with what seems to work on rawhide.